tests/test_basic.py

```diff
@@ -185,6 +185,20 @@ def test_osd_number(rook_cluster):
 @pytest.mark.xfail(reason="This is currently failing due to "
                           "https://github.com/rook/rook/issues/6214")
 def test_add_node(rook_cluster):
+    # patch deployment to reduce ROOK_DISCOVER_DEVICES_INTERVAL from 60m to 2m
+    logger.info("Patching deployment to reduce ROOK_DISCOVER_DEVICES_INTERVAL")
+    patch = {'spec':
+             {'template':
+              {'spec':
+               {'containers':
+                [{'name':
+                  'rook-ceph-operator',
+                  'env': [
+                      {'name': 'ROOK_DISCOVER_DEVICES_INTERVAL', 'value': '2m'}
+                  ]}]}}}}
+    rook_cluster.kubernetes.kubectl(
```

Comment:
no need to use the `patch` command. Just do `-n rook-ceph set env deployment/rook-ceph-operator ROOK_DISCOVER_DEVICES_INTERVAL=2m`

Reply:
ok noted

Reply:
I created a new PR (see https://github.com/SUSE/rookcheck/pull/197) with just that. I think this is useful in any case for our CI.